content/resources/user/using-api.md

```diff
@@ -153,4 +153,49 @@ pip install gen3
 
 # To clone and develop the source:
 git clone https://github.com/uc-cdis/gen3sdk-python.git
-```
\ No newline at end of file
+```
+
+* * *
+# Querying and Downloading Metadata using the API
+* * *
+Users with read access to a project can download individual metadata records in the project or all records in a specified node of the project using the API.
+
+The API endpoint for downloading all the records in a single node of a project is:
+```
+{commons-url}/api/v0/submission/{program}/{project}/export/?node_label={node}&format={json/tsv}
+ Where:
+{commons-url} is the gen3 data commons url (for example, 'data.bloodpac.org'),
+{program} is the program name (for example, 'bpa'),
+{project} is the project name
+{node} is the name of the node
+{json/tsv} is the format in which data will be downloaded, either json or tsv
+ ```
+For example, submitting the following API request will download all the records in the 'sample' node of the project 'training-example' in an example data commons (data.mycommons.org) as a tab-separated values file (TSV):
+```
+ https://data.mycommons.org/api/v0/submission/training/example/export/?node_label=sample&format=tsv
+```
+
+ The API endpoint for downloading a single record in a project is as follows:
+```
+{commons-url}/api/v0/submission/{program}/{project}/export?ids={ids}&format={json/tsv}
+ Where:
+{commons-url} is the gen3 data commons url (for example, 'data.bloodpac.org'),
+{program} is the program name (for example, 'bpa'),
+{project} is the project name
+{ids} is a comma separated list of the UUIDs for the records to be downloaded
```

Comment:
a minor thing: I think we are using `GUID` and `UUID` interchangeably? If so, can we be consistent on that by using maybe `GUID` only? Since it is easier to pronounce

Reply:
👍